### PR TITLE
User can view navigation links.

### PR DESCRIPTION
**Pull request by alvinscheng**
screen shot 2017-06-02 at 3 37 40 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,13 +10,28 @@
     <!-- Bootstrap -->
     <!-- Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous" />
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous" />
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (victormejia) [inline index.html]: delete the theme and the js file references

Comment (alvinscheng) [reply index.html]: Done!

File: styles.css [hunk 2]
```diff
@@ -6,7 +6,40 @@ body {
...
+
+.menu-right {
+  float: right;
+}
+
+.nav-link-text {
+  position: relative;
+  display: inline-block;
+  vertical-align: top;
+  font-family: Roboto, sans-serif;
```
Comment (victormejia) [inline styles.css:39]: looks like you are trying to use a custom font. Grab the Roboto font from Google Fonts.

**Review by victormejia — CHANGES_REQUESTED** [review]

File: styles.css (hunk 2 above)
Comment (alvinscheng) [reply styles.css:39]: Done!

**Pull request merged**